internal/cmd/test.go

```diff
@@ -135,3 +137,43 @@ func runCMD(mainFunc func(), args []string) (err error) {
 
 	return err
 }
+
+type CommonCMDTestCases struct {
+	ExpectedExitCode int
+	ExpectedStdout   string
+	ExpectedStderr   string
+}
+
+func RunCommonCMDTests(test *testing.T, mainFunc func(), args []string, commonCases CommonCMDTestCases, prefix string) (string, string, int) {
+	stdout, stderr, exitCode, err := RunCMDTest(test, mainFunc, args)
+	if err != nil {
+		test.Errorf("%sCMD run returned an error: '%v'.", prefix, err)
+	}
+
+	if len(stderr) > 0 && commonCases.ExpectedStderr == "" {
+		test.Errorf("%sCMD has content in stderr: '%s'.", prefix, stderr)
+	}
```

Comment:
I mentioned this in my previous round.

Why do this check when we can just do an equality check on stderr?
If we expect stderr to by empty, we will pass an empty string
(which will effectively to the same check with less cases).